Stratis.Bitcoin.Features.LightWallet/LightWalletFeature.cs

```diff
@@ -134,7 +137,10 @@ public static IFullNodeBuilder UseLightWallet(this IFullNodeBuilder fullNodeBuil
                         services.AddSingleton<IWalletSyncManager, LightWalletSyncManager>();
                         services.AddSingleton<IWalletTransactionHandler, WalletTransactionHandler>();
                         services.AddSingleton<IWalletManager, WalletManager>();
-                        services.AddSingleton<IWalletFeePolicy, LightWalletFeePolicy>();
+                        if (isStratis)
+                            services.AddSingleton<IWalletFeePolicy, LightWalletFixedFeePolicy>();
+                        else
+                            services.AddSingleton<IWalletFeePolicy, LightWalletFeePolicy>();
```

Comment:
Currently LightWalletFeePolicy is actually a bitcoin only fee policy and is collecting the fee from an external source.
I propose we change the name to LightWalletBitcoinFeePolicy or LightWalletBitcoinExternalFeePolicy

Reply:
I changed it to LightWalletBitcoinExternalFeePolicy